src/pages/google-map/google-map.ts

```diff
@@ -1,13 +1,13 @@
 import { Component, ViewChild } from '@angular/core';
-import { IonicPage, NavController, NavParams} from 'ionic-angular';
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
 
 /**
  * Generated class for the GoogleMapPage page.
  *
  * See https://ionicframework.com/docs/components/#navigation for more info on
  * Ionic pages and navigation.
  */
-
+declare var google; //add it to work in my laptop, feel free to remove it. Xiao
```

Comment:
We'll figure this out when we meet

Reply:
It doesn't break anything on my laptop so shouldn't break on others also, I guess we can just keep it in